backend/prod.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eu
+pip -q install --upgrade pip
+pip -q install --cache-dir=.pip -r requirements.txt
+
+# Run the FastAPI server in port 8000
+fastapi run main.py
```

Comment:
Please use environment variable for port.

Reply:
we don't publish back-end port. So it makes easier for front-end to manage the [proxy config](https://github.com/akvo/rag-doll/blob/54b5a016abf0e90c33a861457a9f6007a65abd4e/frontend/next.config.mjs#L12) while in dev.

Reply:
no worries, I think it's better to have all ports in .env. I'll move it

Reply:
Done